index.html

```diff
@@ -4,9 +4,34 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <title>Portfolio</title>
   </head>
   <body>
-    <h1>I'm Software Developer</h1>
+    <div class="flex-container">
+      <header>
+        <nav>
+          <div class="logo">Welcome</div>
+            <label>
+              <i class="fas fa-bars"></i>
+            </label>
+        </nav>
+      </header>
+      <section>
+        <h1 class="developer-name">Hi There. I'm Mudasir Sherwani</h1>
+        <h2 class="software-developer">I'm Software Developer</h2>
+          <p class="banner-text-pera">I can help you build a product, feature or website Look through some of my 
+            work and experience! if you like what you see and have a project you need coded, don't hasitate to 
+            contact me.
+          </p>
+          <div class="icon">
+            <i class='fab fa-facebook' ></i>
+            <i class='fab fa-instagram'></i>
+            <i class='fab fa-twitter-square'></i>
+            <i class='fab fa-github' ></i>
+            <i class='fab fa-linkedin' ></i>
+            </div>
```

Comment:
- [ ] [OPTIONAL] Please consider putting these icons into list tags. In addition, each icon is a link. It is wrapped into an `<a>`. I would recommend you to work on it even if it is optional. Below is a hint 💡 
```bash
<ul>
<li><a href='#' ><i class='fab fa-facebook' ></i></a></li>
//Other links follows
</ul>
```